### DISCUSSION TITLE
Hide items in the media controller mode

### DISCUSSION THREAD

**Discussion by Uilsonjw** [sircharlo/meeting-media-manager#348] [Ideas] [answered]
During a meeting would be good if we have an option to hide items that already was played in that meeting. Of course, if we hide an item we should be able to show it again. Or another option also aceptable is to have the ability to change the color of the already played item, somithing like put it on gray.

It meke the operation easier when we have a lot of itens in a meeting.

Thanks for take your time in consider it.

**Comment by sircharlo** [comment, marked as answer]
I'm actually working on something like this as we speak!

![image](https://user-images.githubusercontent.com/1275666/169313501-456240cd-46a5-4d56-870e-c80276aab1cd.png)

**Comment by Uilsonjw** [comment]
Great. That is a good news.
